receiver/hostmetricsreceiver/internal/scraper/virtualmemoryscraper/virtualmemory_scraper_windows.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
+
+package virtualmemoryscraper
+
+import (
+	"context"
+	"math"
+	"time"
+
+	"go.opencensus.io/trace"
+
+	"go.opentelemetry.io/collector/component/componenterror"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/receiver/hostmetricsreceiver/internal/windows/pdh"
+)
+
+const (
+	pageReadsPerSecPath  = `\Memory\Page Reads/sec`
+	pageWritesperSecPath = `\Memory\Page Writes/sec`
+)
+
+// scraper for VirtualMemory Metrics
+type scraper struct {
+	config *Config
+
+	pageReadsPerSecCounter  pdh.PerfCounterScraper
+	pageWritesPerSecCounter pdh.PerfCounterScraper
+
+	startTime            pdata.TimestampUnixNano
+	prevPagingScrapeTime time.Time
+	cumulativePageReads  float64
+	cumulativePageWrites float64
+}
+
+// newVirtualMemoryScraper creates a set of VirtualMemory related metrics
+func newVirtualMemoryScraper(_ context.Context, cfg *Config) *scraper {
+	return &scraper{config: cfg}
+}
+
+// Initialize
+func (s *scraper) Initialize(_ context.Context) error {
+	s.startTime = pdata.TimestampUnixNano(uint64(time.Now().UnixNano()))
```

Comment:
Note since we are deriving the counter from per second values, the counter essentially starts from when we take the first reading, thus why I set this to the current time here. I believe that's correct?